scss/_sphinx_layout.scss

```diff
@@ -147,10 +147,40 @@
   }
 }
 
+.pytorch-left-nav-collapsible {
+  display: block;
+  background-image: url(../images/chevron-down-orange.svg);
```

Comment:
I don't think this asset exists. I don't see the image showing up, and it's not in the images folder. We'll probably need Pete to create it for us.